ansible/rke2/airgap/roles/rke2_tarball/tasks/verify_checksums.yml

```diff
@@ -0,0 +1,133 @@
+---
```

Comment:
I don't know that we'll ever really need to verify checksums. It's not incorrect or bad to have this, but just a note that it might be superfluous.